src/main/java/org/folio/edge/core/utils/OkapiClient.java

```diff
@@ -52,6 +54,12 @@ public OkapiClient(OkapiClient client) {
     initDefaultHeaders();
   }
 
+  public OkapiClient(OkapiClient client, String secondaryTenantId) {
+    this(client);
+    this.secondaryTenantId = secondaryTenantId;
+    defaultHeaders.set(X_OKAPI_TENANT, StringUtils.isNullOrEmpty(secondaryTenantId) ? tenant : secondaryTenantId);
```

Comment:
If the argument `secondaryTenantId` is null or empty there will now be a 2 entries in the `MultiMap` with the same value. Is that what we want? Seems wrong.

Wouldn't it better to not add the item if the `secondaryTenantId` param is empty? You could also throw an `IllegalArgumentException` if that is what you are worried about, although that might be more trouble than it is worth. But you don't want to create bad data.